packages/salesforcedx-utils-vscode/src/types/index.ts

```diff
@@ -0,0 +1,41 @@
+/*
```

Comment:
I moved all the command type declarations into salesforcedx-utils-vscode so we can share them.